.gitignore

```diff
@@ -26,3 +26,4 @@ node_modules
 
 # Users Environment Variables
 .lock-wscript
+.idea
```

Comment:
My editor, Webstorm, makes an .idea folder with various config files. Gotta ignore it!